VirtoCommerce.Storefront/Filters/ForceLoginAuthorizeFilter.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Authorization.Policy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Authorization;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.DependencyInjection;
+using VirtoCommerce.Storefront.Domain.Security;
+
+namespace VirtoCommerce.Storefront.Filters
+{
+    /// <summary>
+    /// Authorization filter that redirects all unauthorized users to Login page (deafult AuthorizeFilter could show AccessDenied for non-authorized authenticated users)
+    /// </summary>
+    public class ForceLoginAuthorizeFilter : IAsyncAuthorizationFilter
```

Comment:
This is AuthorizationFilter, not AuthorizeFilter.
The name should correspond to AuthorizationHandler above.

Reply:
Done